Content.Shared/Chemistry/Prototypes/InjectorModePrototype.cs

```diff
@@ -0,0 +1,158 @@
+﻿using Content.Shared.Chemistry.Components;
+using Content.Shared.FixedPoint;
+using Robust.Shared.Audio;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization;
+
+namespace Content.Shared.Chemistry.Prototypes;
+
+[Prototype("InjectorMode"), Serializable]
```

Comment:
We don't need to provide the name of the prototype since it gets inferred (old code still does this), and prototypes aren't serialized.

```suggestion
[Prototype]
```